src/components/ContentHighlights/ContentHighlightArchivedAlert.jsx

```diff
@@ -0,0 +1,49 @@
+import { Alert } from '@edx/paragon';
+import { useContext, useState } from 'react';
+import { NEW_ARCHIVED_CONTENT_ALERT_DISMISSED_COOKIE_NAME } from './data/constants';
+import { EnterpriseAppContext } from '../EnterpriseApp/EnterpriseAppContextProvider';
+import { enterpriseCurationActions, isArchivedContent } from '../EnterpriseApp/data/enterpriseCurationReducer';
+
+const ContentHighlightArchivedAlert = () => {
+  const archivedContentCookies = {};
+  const [isOpen, setIsOpen] = useState(true);
```

Comment:
[suggestion, sanity-check] I might recommend treating this `Alert` as a "controlled" component, whereby it accepts an `open` boolean prop to pass to the `Alert` component and instead defines the `const [isArchivedAlertOpen, setIsArchivedAlertOpen] = useState(false);` in `CurrentContentHighlightHeader`. I could imagine seeing a `useEffect` in `CurrentContentHighlightHeader` that executes when `isNewArchivedContent` changes to adjust the `isArchivedAlertOpen` state variable accordingly.

The primary rationale for this is because, as is, the `Alert` component is already open when its rendered in the DOM for the first time, whereby the fade-in transition/animation wouldn't work as expected. Typically, the `Alert` component should still be rendered, even if not opened initially.

The closing of the `Alert` could be done by passing in an `onClose` prop where a callback function is passed from the parent component, with a state update to `isArchivedAlertOpen`.